bin/entrypoint-cron.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -e
+
+mkdir /var/run/celery/
+
+celery -A rhub.api.celery_worker:celery beat --schedule=/var/run/celery/celerybeat-schedule --loglevel $([ $FLASK_ENV == 'development' ] && echo 'DEBUG' || echo 'INFO')
```

Comment:
Would be better to load the logging config from file `$LOG_CONFIG` and fall back to `$LOG_LEVEL`.

Reply:
I couldn't find an easy way to pass the configuration file to celery CLI.
As we have not defined that celery will be the final solution, can I create a JIRA to address this if we decide to go forward with celery?

Reply:
https://docs.celeryq.dev/en/latest/userguide/signals.html#setup-logging

Yes, JIRA for logging config file is OK, but at least use the LOG_LEVEL variable. I don't want to see debug logs just because I enabled flask development mode.

Reply:
And you have a bug in the condition, if FLASK_ENV is empty or undefined: `bash: [: ==: unary operator expected`